core/audio/start.sh

```diff
@@ -86,4 +86,9 @@ if [[ -n "$SOUND_ENABLE_SOUNDCARD_INPUT" ]]; then
   route_input_source
 fi
 
-exec pulseaudio --file /etc/pulse/balena-sound.pa
\ No newline at end of file
+exec pulseaudio \
+  --realtime=true \
```

Comment:
We are already using `--realtime` though it's not easy to tell :P

If you take a look at the PulseAudio config script [here](https://github.com/balenablocks/audio/blob/master/daemon.conf#L12-L13), that should be equivalent to the realtime flag. If you say latency errors are gone then perhaps there is something wrong with the setup that's worth investigating. The documentation definitely suggest a few caveats to this flags if you are not using the system flag, so that might be the case. 

How confident are you that the latency problems are gone? Also, just so we are on the same page, what latency problems? Error logs would help here :)

Reply:
I cannot reproduce the latency errors I saw previously. But, it's worth noting that now my multiroom client and server services are down, and that might have to do something with it. I can't at the moment debug exactly what have I done that made them down, and TBH I don't need them at all as I have only 1 sink in use. I'm running my own fork for a while now, with changes I don't know if can be upstream.

Reply:
Okay, worth investigating then. I'll try to squeeze it in this week!